workforceDashboard/src/main/java/gov/homeoffice/workforceDashboard/service/EmployeeServiceImpl.java

```diff
@@ -24,4 +30,112 @@ public void saveEmployee (Employee employee){
+                String operationalLineManagerRoleReference = row.getCell(25).getStringCellValue();
+                String currentResourceType = row.getCell(26).getStringCellValue();
+                String targetResourceType = row.getCell(27).getStringCellValue();
+                String projectedStartDateOfRole = row.getCell(28).getStringCellValue();
+                String projectedEndDateOfTerminatingRole = row.getCell(29).getStringCellValue();
+                Double fTE = row.getCell(30).getNumericCellValue();
+                String dDaTOrNonDDaTResource = row.getCell(31).getStringCellValue();
+
+                employee.setRoleRef(roleRef);
+                employee.setRoleTitle(roleTitle);
+                employee.setEmployeeFirstName(employeeFirstName);
+                employee.setEmployeeSurname(employeeSurname);
+                employee.setEmployeeFullName(employeeFullName);
+                employee.setEmployeeAdelphiNumber(employeeAdelphiNumber);
+                employee.setEmployeeEmail(employeeEmail);
+                employee.setGradeEquivalent(gradeEquivalent);
+                employee.setFunction(function);
+                employee.setBusinessUnit(businessUnit);
+                employee.setTeam(team);
+                employee.setProfessionCluster(professionCluster);
+                employee.setdDaTProfessionRole(dDaTProfessionRole);
+                employee.setAffiliatedDdaTProfessionRole(affiliatedDdaTProfessionRole);
+                employee.setCurrentResourceRollUp(currentResourceRollUp);
+                employee.setFunctionRollUp(functionRollUp);
+                employee.setEmployeeCurrentPrimaryLocation(employeeCurrentPrimaryLocation);
+                employee.setEmployeeAnticipatedFutureLocation(employeeAnticipatedFutureLocation);
+                employee.seteUExit(eUExit);
+                employee.setIsThisRoleAVacancy(isThisRoleAVacancy);
+                employee.setVacancyType(vacancyType);
+                employee.setVacancyStage(vacancyStage);
+                employee.setRechargedRoles(rechargedRoles);
+                employee.setOperationalLineManagerFirstName(operationalLineManagerFirstName);
+                employee.setOperationalLineManagerSurname(operationalLineManagerSurname);
+                employee.setOperationalLineManagerRoleReference(operationalLineManagerRoleReference);
+                employee.setCurrentResourceType(currentResourceType);
+                employee.setTargetResourceType(targetResourceType);
+                employee.setProjectedStartDateOfRole(projectedStartDateOfRole);
+                employee.setProjectedEndDateOfTerminatingRole(projectedEndDateOfTerminatingRole);
+                employee.setfTE(fTE);
+                employee.setdDaTOrNonDDaTResource(dDaTOrNonDDaTResource);
+
+                System.out.println("Employee record saved--" + employee.toString());
+                saveEmployee(employee);
+
+            } file.close();
```

Comment:
We should be closing the workbook as well after processing it

Reply:
Completed.  See latest commit